docs/trino-connector/trino-connector.md

```diff
@@ -7,11 +7,13 @@ This software is licensed under the Apache License version 2."
 ---
 
 Trino can manage and access data using the Trino connector provided by `Gravitino`, commonly referred to as the `Gravitino connector`.
-After configuring the Gravitino connector in Trino, it can automatically load catalog metadata from Gravitino, allowing users to directly access these catalogs in Trino.
+After configuring the Gravitino connector in Trino, Trino can automatically load catalog metadata from Gravitino, allowing users to directly access these catalogs in Trino.
 Once integrated with Gravitino, Trino can operate on all Gravitino data without requiring additional configuration.
 
-When metadata such as catalogs, schemas, or tables change in Gravitino, Trino can also update itself through Gravitino.
-
+:::node
+Once metadata such as catalogs, schemas, or tables are changed in Gravitino, Trino can update itself through Gravitino immediately, this process usually takes 
```

Comment:
Actually you need to wait for 3-10 seconds, so it is not proper to say "immediately" here.

Reply:
OK